app/uk/gov/hmrc/hec/models/TaxDetails.scala

```diff
@@ -28,7 +28,8 @@ object TaxDetails {
     sautr: Option[SAUTR],
     taxSituation: TaxSituation,
     saIncomeDeclared: Option[YesNoAnswer],
-    saStatusResponse: Option[SAStatusResponse]
+    saStatusResponse: Option[SAStatusResponse],
+    relevantIncomeTaxYear: Int
```

Comment:
This could be of type `TaxYear` to make it more strongly typed

Reply:
oh yes 🤦‍♀️ , forgot about it

Reply:
updated [here](https://github.com/hmrc/hec/pull/45/commits/d4742c78c333f18cfcda7f0e44d418d8b8aea81d)